graphql_compiler/macros/macro_edge/helpers.py

```diff
@@ -97,6 +99,33 @@ def remove_directives_from_ast(ast, directive_names_to_omit):
     return new_ast
 
 
+def find_target_and_copy_path_to_it(ast):
+    """Return the same ast and the ast at the target, with the path to the target shallow copied."""
```

Comment:
Mind describing in a bit more detail what this function is intended to accomplish? I think I get what it is doing, but not why it's doing it. "Same AST" is also a bit misleading.

Also, a nit: I'd feel better about it with `ast` -> `AST` for consistency.